sql/updates/world/3.3.5/2016_06_15_99_world.sql

```diff
@@ -0,0 +1,40 @@
+-- Fix requirement quest for the alliance
+UPDATE `quest_template_addon` SET `PrevQuestID`=2278, `NextQuestID`=2439 WHERE `ID`=2279;
+
+-- Add gossip text to menu
+
+UPDATE `creature_template` SET `gossip_menu_id`=57000 WHERE `entry`=7918;
+DELETE FROM `gossip_menu` WHERE `entry` IN (57000,57001,57002,57003,57004,57005,57006);
+INSERT INTO `gossip_menu` (`entry`,`text_id`) VALUES 
+(57000,1674),(57001,1675),(57002,1676),(57003,1677),(57004,1678),(57005,1679);
+
+DELETE FROM `gossip_menu_option` WHERE (`menu_id` BETWEEN 57000 AND 57005) OR (`menu_id` IN (8072));
+INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`) VALUES
+(57000,0,0,'What function do you serve?',4054,1,3,57001,0,0,0,'',0),
```

Comment:
OK. travis-ci does not look into the SQL files, so manual verification is needed.